src/test/java/lotto/model/WinningResultsTest.java

```diff
@@ -0,0 +1,57 @@
+package lotto.model;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class WinningResultsTest {
+    @DisplayName("3000원으로 4개를 맞춘 로또가 있으면 수익률이 16.66이 나와야한다.")
+    @MethodSource("provideLottos")
+    @ParameterizedTest
+    void calculateReturnRatio(List<Lotto> provideLottos) {
+        Money money = new Money(3_000);
+        Lotto lotto = new Lotto(
+                List.of(1, 2, 3, 4, 5, 6).stream()
+                        .map(LottoNumber::new)
+                        .collect(Collectors.toList())
+        );
+        LottoNumber bonusNumber = new LottoNumber(7);
+        WinningLotto winningLotto = new WinningLotto(lotto, bonusNumber);
+        Lottos lottos = new Lottos();
+        for (Lotto provideLotto : provideLottos) {
+            lottos.add(provideLotto);
+        }
+        WinningResults winningResults = new WinningResults(winningLotto, lottos);
+        winningResults.calculateWinningResults();
+        Assertions.assertThat(money.calculateReturnRatio(winningResults.calculateEarnedMoney())).isEqualTo(16.66);
```

Comment:
```suggestion
        assertThat(winningResults.calculateEarnedMoney()).isEqualTo(XXXXX);
```
money가 earningRatio를 계산하는건 MoneyTest에서 이뤄졌으니 winningResults가 벌어들인 금액을 검증하면 될 것 같아요!
WinningResults 객체에 대한 단위테스트니까요!

Reply:
앗 그러네요!